src/openvswitch.h

```diff
@@ -29,3 +29,6 @@ netplan_netdef_write_ovs(
 
 NETPLAN_INTERNAL gboolean
 netplan_ovs_cleanup(const char* rootdir);
+
+NETPLAN_INTERNAL const char *
```

Comment:
We might want to drop the `NETPLAN_INTERNAL` attribute here. It will make the `netplan_openvswitch_ovs_vsctl` symbol visible as part of the library, which we don't need here, it should be part of all the relevant object files. (Yes, the NETPLAN_INTERNAL naming is confusing here, sorry for that.)

Should it be needed, the new symbol should be prefixed with "_" (`_netplan_openvswitch_ovs_vsctl`) to clearly mark it as private API.